script/buildSQLGeom.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+
+"""
+This module will accept (x,y,[z]) coordinates (from CSV-format file or cmd arguments)
+and generate a SQL string that will add these coordinates as a PostGIS Polygon
+to sites.geometry within a BETYdb instance.
+
+A header row should be included in the input file, although the column names do not matter. If a z column
+for altitude is not included, the script will attempt to get it from USGS NED (US) or Google (global).
+
+Command line usage:
+        -h      show help
+        -i      input filename; will override x,y,z args if provided
+        -x      longitude value, if no file provided
+        -y      latitude value, if no file provided
+        -z      altitude value, if no file provided
+        -r      EPSG reference number, default is 4326
+        -s      sitename or site_id in BETYdb sites table to associate geometry with.
+                if a string is provided, site_id will be queried. if an int is provided, site_id is assumed.
+        -o      flag to write resulting SQL to output file, as "<input_filename>_insert.sql"
+        -e      will attempt to execute resulting SQL on db instance defined in next 4 params
+        -n      PostgreSQL host address
+        -d      Database name, default is 'bety'
+        -u      Username
+        -p      Password
+
+Examples:
+        buildSQLGeom.py -i C:/folder/coordinates.csv -r 4326 -s 19000000000 -e -n localhost -d bety -u GUEST -p GUES -o
+        buildSQLGeom.py -x 76.116081 -y 42.794448
```

Comment:
Currently, if not site_id or sitename is provided, site_id is set to 0. I can modify it to create a new site, but from what I understand of [the sites table](https://www.betydb.org/db_docs/tables/sites.html) we would need these non-nullable fields:
* city
* state
* country
* soil
* notes
* soilnotes
* sitename

I'll augment the examples to include one of those arguments.